ipython/dns/UI.md

```diff
@@ -0,0 +1,592 @@
+# Open Network Insight - Operational Analytics User Interface
+
+Provides tools for interactive visualization, noise filters, white listing, and attack heuristics.
+
+## Table of Content
+
+- [Open Network Insight - Operational Analytics User Interface](#open-network-insight---operational-analytics-user-interface)
+  * [Table of Content](#table-of-content)
+  * [Intended Audience](#intended-audience)
+  * [Getting Started](#getting-started)
+  * [Technical Documentation](#technical-documentation)
```

Comment:
I would put the build, run, debug information before the lengthy discussion of the guts of each component